app/src/main/AndroidManifest.xml

```diff
@@ -63,7 +63,7 @@
             android:name=".activity.MainActivity"
             android:exported="true"
             android:label="@string/app_name"
-            android:launchMode="singleTop"
+            android:launchMode="singleInstance"
```

Comment:
Why is this change being made? We do not want multiple instances of MainActivity floating around.

Reply:
I didn't change it, this change from PR #2176.